OracleDatabase/RAC/OracleRealApplicationClusters/dockerfiles/12.2.0.1/MultiNodeInstall.py

```diff
@@ -0,0 +1,310 @@
+#!/usr/bin/python
+#!/usr/bin/env python
+
+###########################################################################################################
+
+
+# stig.py
+#
+# Copyright (c) 2012, 2018, Oracle and/or its affiliates. All rights reserved.
+#
+# NAME
+#      buildImage.py - <Build the Image>
+#
+# DESCRIPTION
+#      <This  script will build the Image>
```

Comment:
This is not a great description. What image? What build? What are the parameters? If you're going to have a description, please make it descriptive. I see a bunch of parameters and options lower in the script, so those should be documented here as well.

Reply:
Added more details